.circleci/config.yml

```diff
@@ -285,6 +285,13 @@ workflows:
       - unit-test
       - docker-build-push:
           context: aws-dev
+          filters:
+            branches:
+              only:
+                - main
+                - auto
+                - canary
```

Comment:
Can we add a testnet branch too? Thanks!

Reply:
done

Reply:
Let’s keep both of them, testnet and devnet are both in use now